Lab1.ipynb

```diff
@@ -241,15 +249,256 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n = 10\n",
+    "p = .9\n",
+    "# defining list of r values \n",
+    "r_values = list(range(n + 1)) \n",
+    "# list of pmf values \n",
+    "dist = [binom.pmf(r, n, p) for r in r_values ] \n",
+    "# plotting the graph  \n",
+    "plt.bar(r_values, dist) \n",
+    "plt.xlabel('X', fontsize = 27) \n",
+    "plt.ylabel('P(x)', fontsize = 27)\n",
+    "plt.title('Bionomial Distribution w/ n = 10, p = .9', fontsize = 30)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Seen in the code above with a binomial distrbution, it's almost impossible to succeed less than five times in a 10 trial grouping. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "C: Does a greater sample size of 15 have a larger expetected value than a sample size of 10?"
```

Comment:
Part 2 Problem 1C:
- you need a question that relates to statistical probability of a binomial distribution: from the lab instructions, "Again define a hypothetical question (clearly stated), and calculate the corresponding probability". This does not search for a probability?
- I think our proposed questions should be similar to the one from Part 1 Problem 3B, but this time we are working with discrete data
- maybe try asking a question for a binomial distribution with known n and p and searching for a probability of a certain n number of events occurring? for example, in your last graph, try calculating the probability of success of x=8 events when p=0.9 and n=10 trials?